src/ProjectService.Models.Db/DbProject.cs

```diff
@@ -40,6 +40,14 @@ public void Configure(EntityTypeBuilder<DbProject> builder)
             builder
                 .HasKey(p => p.Id);
 
+            builder
+                .Property(P => P.Name)
+                .IsRequired();
+
+            builder
+                .Property(p => p.ShortName)
+                .IsRequired();
+
```

Comment:
I think you need to add conditions to DepartmentId and Status.

Reply:
this are structure